inst/EpiModel2.rmd

```diff
@@ -56,13 +63,13 @@ target.stats <- c(30, 0)
 
 It is important to note, that the vertex attribute "group" now takes on a special meaning within the context of EpiModel modeling, separate from other attribute names. Users should be mindful when constructing networks in which "group" is a vertex attribute and note whether or not the interpretation of group aligns with that detailed above.
```

Comment:
also have suggestions for line 58 that I am unable to make a suggested change for here:

"In this network we have a total of 30 edges (or sexual relationships) in the population, two of which involve individuals of the same sex. We can replicate the traditional bipartite behavior of no within-group mixing by setting the target statistic for total number of within-group edges to 0:"

I also added `` to "group" in the suggested changes below to be consistent with how you had it above, but feel free to change all mentions of "group" (including above) to double quotes "" if more appropriate

```suggestion
It is important to note that the vertex attribute `group` now takes on a special meaning within the context of EpiModel, separate from other attribute names. Users should be mindful when constructing networks in which `group` is a vertex attribute and note whether or not the interpretation of `group` aligns with that detailed above.
```